connector/src/main/java/org/wso2/carbon/connector/salesforce/SetupConvertLead.java

```diff
@@ -0,0 +1,51 @@
+/*
+ *  Copyright (c) 2016, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+ *
+ *  WSO2 Inc. licenses this file to you under the Apache License,
+ *  Version 2.0 (the "License"); you may not use this file except
+ *  in compliance with the License.
+ *  You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+package org.wso2.carbon.connector.salesforce;
+
+import org.apache.synapse.MessageContext;
+import org.apache.synapse.SynapseLog;
+import org.wso2.carbon.connector.core.AbstractConnector;
+
+public class SetupConvertLead extends AbstractConnector {
+
+    public void connect(MessageContext synCtx) {
+
+        SynapseLog synLog = getLog(synCtx);
+
+        if (synLog.isTraceOrDebugEnabled()) {
+            synLog.traceOrDebug("Start : Salesforce Find Duplicates mediator");
+
+            if (synLog.isTraceTraceEnabled()) {
+                synLog.traceTrace("Message : " + synCtx.getEnvelope());
+            }
+        }
+
+        SalesforceUtil salesforceUtil = SalesforceUtil.getSalesforceUtil();
+        salesforceUtil.addSobjects("convertLead", SalesforceUtil.SALESFORCE_LEAD_CONVERT, synCtx, synLog,
+                null);
+
+        if (synLog.isTraceOrDebugEnabled()) {
```

Comment:
Is this two nested related to logs are common? What is the difference between the two? I am thinking of simplifying and moving to a common method.

Reply:
To indicate start and end of the class mediator operation. One for start and another for end

Reply:
+1. We will add it to connector improvement doc.